### PR TITLE
Styling cleanup

### LINKED ISSUE #102: Service card view duplicates tasks
#### Steps to reproduce the issue

1.  Navigate to services card view
2. Click on service card
3. Close card
3. Click on same card again

#### What's the expected result?
- The tasks displayed do not change unless the scale has also changed

#### What's the actual result?
- Tasks associated with the service are duplicated in the view

#### Additional details / screenshot

- screen shot 2018-05-12 at 14 34 30

### LINKED ISSUE #105: Volumes card view modal dialogue appears as a card
#### Steps to reproduce the issue
1.  Navigate to volume card view
2. Click on card

#### What's the expected result?
- A modal dialogue as with the Task Card View

#### What's the actual result?
- The dialogue is added as a card to the bottom of the card view

#### Additional details / screenshot
volume_modal

### LINKED ISSUE #106: Change the way styling is done

### PR DESCRIPTION
**Pull request by FJMentz**
Call me mr Styles. Updated the whole theme and extracted most css files to a general global file.

Also some other small fixes

----
Fixes #105 
Fixes #102 
Closes #106

### CODE REVIEW

**Review by annamarieHelberg — APPROVED** [review]
Looks good

**Review by ghost — APPROVED** [review]

**Review by egeldenhuys — APPROVED** [review]
Fixes #105 
Fixes #102 
Closes #106

File: src/styles.css [hunk 1]
```diff
@@ -16,67 +16,83 @@
...:center;-ms-flex-align:center;align-items:center;padding:0.375rem 0.75rem;margin-bottom:0;font-size:0.875rem;font-weight:400;line-height:1.5;color:#495057;text-align:center;white-space:nowrap;background-color:#F8F5F0;border:1px solid #ced4da;border-radius:0.25rem}.input-group-text input[type="radio"],.input-group-text input[type="checkbox"]{margin-top:0}.input-group>.input-group-prepend>.btn,.input-group>.input-group-prepend>.input-group-text,.input-group>.input-group-append:not(:last-child)>.btn,.input-group>.input-group-append:not(:last-child)>.input-group-text,.input-group>.input-group-append:last-child>.btn:not(:last-child):not(.dropdown-toggle),.input-group>.input-group-append:last-child>.input-group-text:not(:last-child){border-top-right-radius:0;border-bottom-right-radius:0}.input-group>.input-group-append>.btn,.input-group>.input-group-append>.input-group-text,.input-group>.input-group-prepend:not(:first-child)>.btn,.input-group>.input-group-prepend:not(:first-child)>.input-group-text,.input-group>.input-group-prepend:first-child>.btn:not(:first-child),.input-group>.input-group-prepend:first-child>.input-group-text:not(:first-child){border-top-left-radius:0;border-bottom-le
```
Comment (egeldenhuys) [inline src/styles.css]: Why is the bootstrap table theme included in `styles.css` and `styles-additional.css` ?

**Comment by annamarieHelberg** [comment]
@FJMentz This really looks amazing. But, I do think we should look at having an option where a user can decide between styles (for example, dark theme and this light (official Compiax) theme).

**Review by egeldenhuys — CHANGES_REQUESTED** [review]
Will merge once the duplicate bootstrap table style is removed from style.css

**Review by egeldenhuys — APPROVED** [review]

**Pull request merged**